package.json

```diff
@@ -92,7 +92,7 @@
         "sass-loader": "^7.1.0",
         "semver": "5.5.0",
         "semver-sort": "0.0.4",
-        "styled-components": "^4.1.1",
+        "styled-components": "^5.0.0",
```

Comment:
https://styled-components.com/releases#v5.0.0 (lots of cool stuff + no breaking changes)